README.md

```diff
@@ -27,24 +27,45 @@ Using Ember Exam is fairly straightforward as it extends directly from the defau
 $ ember exam
 $ ember exam --filter='acceptance'
 $ ember exam --server
+$ ember exam --load-balance --parallel=1
+```
+
+A value to an option can be passed with either `=` or a space.
+```bash
+# A value of filter is acceptance
+$ ember exam --filter 'acceptance'
+
+# A value of parallel is 2
+$ ember exam --load-balance --parallel=2 --server --no-launch
+
+# A value of parallel is --server
```

Comment:
I think it would be help to elaborate a bit more here for this case.
```
# If a `=` is not used to pass a value to an option that requires a value, it will take anything passed after a space as it's value
# In this instance, the value of parallel is --server
```